app/ui-message/client/message.css

```diff
@@ -0,0 +1,15 @@
+pre {
+    font-family: -apple-system,
+                 BlinkMacSystemFont,
+                 'Segoe UI',
+                 Roboto,
+                 Oxygen,
+                 Ubuntu,
+                 Cantarell,
+                 'Helvetica Neue',
+                 'Apple Color Emoji',
+                 'Segoe UI Emoji',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 tabs (indentation)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=4929452)